sdk/deepbook-v3/examples/deepbookMarketMaker.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { decodeSuiPrivateKey } from '@mysten/sui.js/cryptography';
```

Comment:
This isn't the right package name.  The fact that this isn't erroring means this file isn't being type-checkin in ci

Reply:
What's the correct package for converting private key?

Reply:
its should be '@mysten/sui instead of '@mysten/sui.js